test/recipes/25-test_crl.t

```diff
@@ -36,6 +36,11 @@ ok(compare1stline([qw{openssl crl -noout -fingerprint -sha256 -in},
                    srctop_file('test', 'testcrl.pem')],
                   'SHA256 Fingerprint=B3:A9:FD:A7:2E:8C:3D:DF:D0:F1:C3:1A:96:60:B5:FD:B0:99:7C:7F:0E:E4:34:F5:DB:87:62:36:BC:F1:BC:1B'));
 
+ok(compare1stline_stdin([qw{openssl crl -hash -noout}],
+                        srctop_file("test","testcrl.pem"),
+                        'issuer name hash=106cd822'),
```

Comment:
This will be broken after #15044  is merged

Reply:
thanks for the update - will change once the PR is merged..

Reply:
I did notice the change in format..

Reply:
Did the change now.. This will mean the tests all fail currently..